common/FileUtil-unix.cpp

```diff
@@ -94,70 +94,43 @@ namespace FileUtil
     }
 #endif
 
-    static int nftw_cb(const char *fpath, const struct stat*, int type, struct FTW*)
-    {
-        if (type == FTW_DP)
-        {
-            rmdir(fpath);
-        }
-        else if (type == FTW_F || type == FTW_SL)
-        {
-            unlink(fpath);
-        }
-
-        // Always continue even when things go wrong.
-        return 0;
-    }
 
-    void removeFile(const std::string& path, const bool recursive)
+ void removeFile(const std::string& path, const bool recursive)
     {
         LOG_DBG("Removing [" << Anonymizer::anonymizeUrl(path) << "] " << (recursive ? "recursively." : "only."));
+        std::error_code err;
 
-        try
-        {
-            struct stat sb;
-            errno = 0;
-            if (!recursive || stat(path.c_str(), &sb) == -1 || S_ISREG(sb.st_mode))
-            {
-                // Non-recursive directories and files that exist.
-                if (errno != ENOENT)
-                    Poco::File(path).remove(recursive);
-            }
-            else
-            {
-                // Directories only.
-                nftw(path.c_str(), nftw_cb, 128, FTW_DEPTH | FTW_PHYS);
-            }
+        if (!recursive) {
+            std::filesystem::remove(path, err);
+        } else {
```

Comment:
Please either use clang-format to automatically format your code, or manually follow the coding style in the file and project. As you can see, all braces are on new lines.

Consistency is great.